### ISSUE TITLE
Incorrect git branch theme colour?

### ISSUE THREAD

**Issue by wildcardjoker** [JanDeDobbeleer/oh-my-posh2#263]
I'm having an issue with the Agnoster theme on a machine. I'm using Windows Terminal and PowerShell, and the git foreground colour is white instead of black using the Agnoster theme.
![image](https://user-images.githubusercontent.com/8613803/83987301-570c2100-a972-11ea-9eb7-3932c4358f7b.png)

The strange thing is that it only does this on one of my machines. Using the same themes, fonts and other settings I get the expected results (black foreground for git status and leading separator bar triangle).
![image](https://user-images.githubusercontent.com/8613803/83987316-65f2d380-a972-11ea-9bbf-f88bcdb92d45.png)

**Comment by SirJson** [comment]
The current color system and how well Themes are staying true to it varies a bit. On top of that the color system is still limited by what Windows defined as Console Colors back then which itself is just a index into your current Terminal theme.

TLDR: I think this is a known issue(?) but needs some refactoring in order to get it right. All dependencies would support such a refactor, someone just needs to properly do it and also maybe talk to the project owner.

**Comment by JanDeDobbeleer** [comment]
When the project started out, this was the only way to tackle colors on Windows. Nowadays we have additional capabilities, especially when using ANSI, but that requires quite the refactor. The good news is that I actually have a new engine in the works that will use hex codes for maximum customization. But that one's not ready for prime time just yet (might need to alpha track it)

**Comment by wildcardjoker** [comment]
Thanks, I thought I may have corrupted something somewhere along the line.
